src/components/Search.js

```diff
@@ -7,35 +9,16 @@ const Search = props => (
     </div>
     <div className="row">
       <div className="col">
-        <table className="table search-table">
-          <thead>
-            <tr>
-              <th>Customer id</th>
-              <th />
-              <th></th>
-              <th />
-            </tr>
-          </thead>
-          <tbody>
-            <tr>
-              <td>
-                <input
-                  id="customerId"
-                  type="text"
-                  className="form-control"
-                  placeholder="Customer id"
-                  name="customerId"
-                />
-              </td>
-              <td>
-                <button className="btn btn-primary fn-submit-name">Search IDs</button>
-              </td>
-              {/* Add search by name here */}
-            </tr>
-          </tbody>
-        </table>
+        { <SearchTable  title="Customer id" placeholder="Customer id" buttonText="Search IDs" id={1}/> }
```

Comment:
The `<SearchTable>` shouldn't be inside `{ }`, there is no need for it.

The `{...}` is for when you want to run javascript code, inside the JSX parts.